docs/tutorials/data_manipulation/pandasdataframes.md.input

```diff
@@ -354,3 +355,62 @@ for index, ax in zip(indices, axes):
     forecasts[index].plot(prediction_intervals=[90.], color='g')
     plt.legend(["observed", "predicted median", "predicted 90% quantile"])
 ```
+
+
+## Use case 2 - Loading data from a single "long" CSV file and filling missing rows
+
+In this section we load data from a long-dataframe format, where each time series is
+stacked on top of each other and has gaps that need to be filled.
+
+To fill the gaps (to include new rows), we group by the `item_id` and reindex the
+dataframe according to our needs. New rows will be created with `NaN`s which
+we fill with -1.
```

Comment:
Is this something one would do? Why not just leaving the `nan`s there? In a sense, reindexing to get the complete time index already "fills the gaps" with `nan`s, then whether `nan`s there make sense or another assumption should be made (like missing data meaning `0`) depends on the context. However `-1` seems a little arbitrary

Reply:
filling -1 for missing values when the rest is positive makes it more visible when printing.. that was the only reason :D . but you are right, no need to fill in any value here.